src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -724,6 +759,17 @@ export class StudyViewPageStore {
         }
     }
 
+    @action
+    async updateUserState(isLoggedIn:boolean) {
```

Comment:
why not just make these observables (loadUserSettingsInitially) into @computed which reference the appStore directly?

Reply:
one could argue that couples the two stores too tightly, but i think it's ok.

Reply:
updated